kalite/distributed/static/js/distributed/contentrec/hbtemplates/content-explore-topic.handlebars

```diff
@@ -3,17 +3,17 @@
   <div class="content-explore-topic">
     <div class="row">
       <div class="col-xs-12">
-          <h4><small>{{_ "Based on your interest in" }} {{ interest_topic.title }}:</small></h4>
+          <p class="up10">{{_ "Based on your interest in" }} <b>{{ interest_topic.title }}</b><span class="sr-only">{{_ ", we recommend you" }}</span>:</p>
       </div>
     </div>
     <div class="row">
       <div class="col-xs-5">
-          <h3>{{ suggested_topic.title }}:</h3>
+          <h2 class="h3">{{ suggested_topic.title }}:</h2>
       </div>
       {{#if suggested_topic.description }}
       <div class="col-xs-7">
-        <h4>{{_ "Description" }}:</h4>
-        <p>{{ truncate suggested_topic.description 70 }}</p>
+        <p class="h4">{{_ "Description" }}:</p>
+        <p>{{ truncate suggested_topic.description 100 }}</p>
```

Comment:
Yeah, I think adding something specifically for screen readers that allows them to read the full description would be best (and even hiding the description for screen readers so that they can read all the titles without interruption).